src/web/ng/impl/WsConnection.hpp

```diff
@@ -57,13 +60,17 @@ class WsConnectionBase : public Connection {
     using Connection::Connection;
 
     virtual std::optional<Error>
```

Comment:
Maybe make it std::expected<void, Error>

Reply:
I added it with this return type to match other methods of `Connection`.  Created an issue for refactoring https://github.com/XRPLF/clio/issues/2292